nym-vpn-desktop/src-tauri/src/error.rs

```diff
@@ -32,6 +32,24 @@ impl CmdError {
     }
 }
 
+impl From<nym_vpn_lib::error::Error> for CmdError {
+    fn from(value: nym_vpn_lib::error::Error) -> Self {
+        CmdError {
+            source: CmdErrorSource::InternalError,
+            message: value.to_string(),
+        }
+    }
+}
+
+impl From<nym_vpn_lib::gateway_directory::Error> for CmdError {
+    fn from(value: nym_vpn_lib::gateway_directory::Error) -> Self {
+        CmdError {
+            source: CmdErrorSource::InternalError,
+            message: value.to_string(),
+        }
+    }
+}
```

Comment:
@octol is all errors from `nym_vpn_lib::gateway_directory` and `nym_vpn_lib::error::Error` can be considered as "internal error" and not "user responsibility" (aka "caller") errors?

Reply:
The calls I've seen used here can only really fail for "internal" reasons, like the endpoint not returning the expected data. The exception I think would be if the API urls are not set correctly, that would probably qualify as a caller error

Reply:
Ok thx
Also, the trait implem for `nym_vpn_lib::error::Error` is not needed and not wanted, as there is plenty of errors from lib calls that can be qualified as "user" related errors
so @zaneschepke lets remove that one